manifests/charts/base/templates/zzz_profile.yaml

```diff
@@ -15,6 +15,7 @@ However, we can workaround this by placing all of (1) under a specific key (.Val
 We can then merge the profile onto the defaults, then the user settings onto that.
 Finally, we can set all of that under .Values so the chart behaves without awareness.
 */}}
+{{- $globals := pick ($.Values.global | default dict) "hub" "tag" "variant" "pollPolicy" }}
```

Comment:
We probably want all of them, since some others like `istioNamespace` are required as well.

or at least audit all ofthem